sqldelight-compiler/build.gradle

```diff
@@ -61,6 +61,10 @@ val VERSION = "${project.version}"
   }
 }
 
-tasks.named('compileKotlin').configure { dependsOn('pluginVersion') }
-
 apply from: "$rootDir/gradle/gradle-mvn-push.gradle"
+
+afterEvaluate {
```

Comment:
I can fix this properly in a follow-up. Basically we just need to register the task to the source set rather than adding its output directory to the source set.